tools/RAiDER/delayFcns.py

```diff
@@ -100,12 +101,12 @@ def get_delays(
     t0 = time.time()
 
     # Get the weather model data
-    with h5py.File(wm_file, 'r') as f:
-        xs_wm = f['x'][()].copy()
-        ys_wm = f['y'][()].copy()
-        zs_wm = f['z'][()].copy()
-        wet = f['wet'][()].copy()
-        hydro = f['hydro'][()].copy()
+    with Dataset(wm_file, mode='r') as f:
```

Comment:
I know the HDF5 version was already here, but I'm thinking to start getting this massaged into a more modular form we should have a generic reader that also takes a file type, and implement both of these bits for a more generic code.